flux/README.md

```diff
@@ -67,10 +67,15 @@ spec:
       - command:
           - /bin/sh
           - -c
-          - mkdir -p /build/plugins && cp -r /plugins/* /build/plugins/
+          - mkdir -p /build/plugins && cp -r /plugins/* /build/plugins/ && chown -R 100:101 /build
         image: ghcr.io/headlamp-k8s/headlamp-plugin-flux:latest
         imagePullPolicy: Always
         name: headlamp-plugins
+        securityContext:
```

Comment:
This part here is required because of the `chown` used above? Or is it for something else?

Reply:
The [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) overwrite the [USER headlamp](https://github.com/headlamp-k8s/plugins/blob/main/Dockerfile#L58C1-L59C1) non root user.
Without it, we don't have enough permissions to do the dirs with non `headlamp` owner.

Reply:
Ok, that makes sense now. Thanks.

Could you please add a comment about that in there for future readers?

Reply:
Yes, it's true. What is the user who run this cmd? 
Without it, you will fail. 
Btw, you can try to run without these changes to check it.

Reply:
> Umm i don't understand this change we already did chwon above so we would not want to run this as root right??
> 
> maybe this is what we need here
> 
> ```
> securityContext:
>   runAsNonRoot: true
>   runAsUser: 100
>   runAsGroup: 101
>   privileged: false
> ```

Again, did you check the root cause of these changes [here ](https://github.com/headlamp-k8s/plugins/issues/214#issuecomment-2987831094)?